provider/ansible.rb

```diff
@@ -269,11 +271,13 @@ def generate_resource_tests(data)
         FileUtils.mkpath target_folder
 
         name = module_name(data[:object])
+        path = File.join(target_folder,
+                         "test/integration/targets/#{name}/tasks/main.yml")
         generate_resource_file data.clone.merge(
           default_template: 'templates/ansible/integration_test.erb',
-          out_file: File.join(target_folder,
-                              "test/integration/targets/#{name}/tasks/main.yml")
+          out_file: path
         )
+        %x(python3 -m black --line-length 160 -S #{path} 2> /dev/null) if path.end_with?('.py')
```

Comment:
Can we have some kind of `run_formatter` function that does the following:

1) Runs the command.
2) If command fails with error, prints error and keeps going.
3) If command doesn't exist, specifically says command doesn't exist and keeps going

Basically, I don't want to have MM users have to install gofmt, black, and rubocop. MM will split out less pretty-code, but it won't crash.

Reply:
Done.  The output code likely won't compile in terraform and likely won't produce valid output in python, but it'll come out anyhow.